### PR TITLE
integrated mongo route and implemented demo function

### PR DESCRIPTION
**Pull request by awilli70**

### CODE REVIEW

File: routes/mongo.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+require('dotenv').config();
+var express = require('express');
+var router = express.Router();
+const MongoClient = require('mongodb').MongoClient;
+const uri = "URIstring"
```
Comment (reshadbinharun) [inline routes/mongo.js:5]: Lets still include this string `const uri = mongodb://${process.env.DBUSER}:${process.env.DBPASSWORD}@${process.env.DBHOST}/${process.env.DB};`
here

File: routes/mongo.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+router.get('/inputone', async (req, res) => {
```
Comment (reshadbinharun) [inline routes/mongo.js:8]: nit: inputOne*

File: routes/mongo.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    try {
+        await client.connect();
```
Comment (reshadbinharun) [inline routes/mongo.js:10]: Are we closing the connection at some point?

**Review by reshadbinharun — APPROVED** [review]

File: routes/mongo.js (hunk 3 above)
Comment (awilli70) [reply routes/mongo.js:10]: Good catch - will update all 3 and submit new PR

**Pull request closed without merging**